.github/workflows/pdf-visual-qa.yml

```
@@ -0,0 +1,198 @@
+    inputs:
+      pdf_path:
+        description: 'Path to PDF to validate'
+        required: false
+        type: string
+
+jobs:
+  visual-qa:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)